src/foam/mlang/MLang.java

```diff
@@ -17,6 +17,8 @@
  */
 public class MLang
 {
+  public static final Predicate TRUE = new True();
```

Comment:
This makes me realize that we have a bug since True and False should be Singletons. I'll fix.